lib/index.js

```diff
@@ -382,6 +429,7 @@ exports.init = (sequelize, optionsArg) => {
 									path: difference.path[0],
 									document,
 									diff,
+									revisionId: revision.id,
```

Comment:
I get a NULL value in column error here. I don't see where the association between revision and change is made tbh without this value